.github/workflows/ci.yml

```
@@ -42,6 +42,7 @@ jobs:
         with:
           name: lint-reports
           path: build/reports/ktlint
+        if: ${{ always() }}
```

Expressions in an if-statement don't require the `${{ }}` syntax
```suggestion
        if: always()
```

---

I don't think this is that big of a deal, I thought I would just let you know!